versions.json

```diff
@@ -1,4 +1,8 @@
 [
+    {
+        "version": "1.39.0",
+        "release_date": "2022-08-29"
+    },
```

Comment:
I _guess_ it doesn't matter anymore, but unsure: there was a version 1.38.1 that never made it into this list

Reply:
@alexkim205 what was the release date for 1.38.1? 16th August? 

Can follow up with an update to add this... the historical record matters :)